Scripts/extras/pull_and_build_from_git.py

```diff
@@ -171,11 +174,18 @@ def _get_value(value_key, required=True, default=None):
         self.package_version = _get_value("package_version")
         self.patch_file = _get_value("patch_file", required=False)
         self.git_commit = _get_value("git_commit", required=False)
-        self.cmake_find_template = _get_value("cmake_find_template")
+        self.cmake_find_template = _get_value("cmake_find_template", required=False)
+        self.cmake_find_source = _get_value("cmake_find_source", required=False)
         self.cmake_find_target = _get_value("cmake_find_target")
         self.cmake_find_template_custom_indent = _get_value("cmake_find_template_custom_indent", default=1)
         self.additional_src_files = _get_value("additional_src_files", required=False)
 
+        if self.cmake_find_template is not None and self.cmake_find_source is not None:
+            raise BuildError("Bad build config file. 'cmake_find_template' and 'cmake_find_template' cannot both be set in the configuration.")            
+        if self.cmake_find_template is None and self.cmake_find_source is None:
+            raise BuildError("Bad build config file. 'cmake_find_template' or 'cmake_find_template' must be set in the configuration.")
```

Comment:
This can be simplified to be `if not self.cmake_find_template and not self.cmake_find_source`
```suggestion
        if self.cmake_find_template and self.cmake_find_source:
            raise BuildError("Bad build config file. 'cmake_find_template' and 'cmake_find_template' cannot both be set in the configuration.")            
        if not self.cmake_find_template and not self.cmake_find_source:
            raise BuildError("Bad build config file. 'cmake_find_template' or 'cmake_find_template' must be set in the configuration.")
```

Reply:
The BuildError strings both say cmake_find_template twice, Both need adjusting so the 2nd name is cmake_find_source.